generators/entity-client/partials/react/entity.tsx.js

```diff
@@ -0,0 +1,37 @@
+const file = context => `${context.webappDir}app/entities/${context.entityFolderName}/${context.entityFileName}.tsx`;
+
+const tmpls = [
+    {
+        type: 'replaceContent',
+        target: context => `<Translate contentKey="${context.baseName}App.${context.entityInstance}.${context.tenantNameLowerFirst}">${context.tenantNameUpperFirst}</Translate>`,
+        tmpl: context => `{!this.props.account.${context.tenantNameLowerFirst} ? <Translate contentKey="${context.baseName}App.${context.entityInstance}.${context.tenantNameLowerFirst}">${context.tenantNameUpperFirst}</Translate> : ''}` // eslint-disable-line
+    },
+    {
+        type: 'replaceContent',
```

Comment:
not sure what these updates are for below? Again, I'm not too clear on React codebase for Jhipster